domains/taitai2017.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "this is my profile",
+  "repo": "https://github.com/taitai2107/web_cv",
+  "owner": {
+    "username": "taitai2107",
+    "email": "nguyencongkhanhtai21072001@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://taitai2107.github.io/web_cv/"
```

CNAME cannot contain protocol or slashes
```suggestion
    "CNAME": "taitai2107.github.io"
```